app.js

```diff
@@ -6,20 +6,40 @@ const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 
 const app = express();
 const punkAPI = new PunkAPIWrapper();
+hbs.registerPartials(__dirname + '/views/partials');
 
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
+
 // Register the location for handlebars partials here:
 
 // ...
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index' );
+});
+
+
+
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => res.render('beers', {beers : beersFromApi}))
+  .catch(error => console.log(error));
 });
 
+app.get('/random-beer', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then(beersFromApi => res.render('random-beer', {beers : beersFromApi}))
```

Comment:
The random api returns an array of one element. You could either pass the array like you did here or you can also pass the beer object.
`{beers : beersFromApi[0]}`